presto-docs/src/main/sphinx/release/release-345.md

```diff
@@ -0,0 +1,57 @@
+# Release 345 (XX Oct 2020)
+
+## General Changes
+
+* Add {func}`concat_ws` function. ({issue}`4680`)
+* Add support for {func}`extract` for `time with time zone` values with precision other than 3. ({issue}`5539`)
+* Add dynamic filtering support for right joins. ({issue}`5555`) 
+* Add support for catalog session properties to session property manager. Both file and database backed property 
```

Comment:
This feature already existed in the engine, so we should say

* Add support for catalog session properties to the file and database backed session property managers.